files/default/event_handler/write_chef_error_handler.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
minor: the copyright should be: 2022 Amazon.com because we created the file this year.

Reply:
fixed